### PR TITLE
Databases week1

### PR DESCRIPTION
**Pull request by alina-kho**
Homework for the 1st week of databases module. Comments and suggestions are higly appreciated ☺️

### CODE REVIEW

File: databases/week1/homework.sql [hunk 1]
```diff
@@ -0,0 +1,95 @@
+-- Task 1 Find out how many tasks are in the task table
+SELECT 
+  COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 2]
```diff
@@ -0,0 +1,95 @@
+-- Task 1 Find out how many tasks are in the task table
+SELECT 
+  COUNT(*) 
+FROM 
+  task;
+
+-- Task 2 Find out how many tasks in the task table do not have a valid due date
+SELECT 
+  COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

**Review by lawansubba — COMMENTED** [review]
Good Work !! 
No major comments, just 2 minor ones

File: databases/week1/homework.sql (hunk 1 above)
Comment (alina-kho) [reply databases/week1/homework.sql]: Fixed :)

File: databases/week1/homework.sql (hunk 2 above)
Comment (alina-kho) [reply databases/week1/homework.sql]: Fixed :) Thank you for the feedback!

**Pull request merged**